plugins/woocommerce/client/blocks/assets/js/blocks/classic-template/archive-product.ts

```diff
@@ -78,23 +74,8 @@ const getDescriptionAllowingConversion = ( templateTitle: string ) =>
 		templateTitle
 	);
 
-const getDescriptionDisallowingConversion = ( templateTitle: string ) =>
-	sprintf(
-		/* translators: %s is the template title */
-		__(
-			'This block serves as a placeholder for your %s. It will display the actual product image, title, price in your store. You can move this placeholder around and add more blocks around to customize the template.',
-			'woocommerce'
-		),
-		templateTitle
-	);
-
-const getDescription = ( templateTitle: string, canConvert: boolean ) => {
-	if ( canConvert ) {
-		return getDescriptionAllowingConversion( templateTitle );
-	}
-
-	return getDescriptionDisallowingConversion( templateTitle );
-};
+const getDescription = ( templateTitle: string ) =>
+	getDescriptionAllowingConversion( templateTitle );
```

Comment:
Seems like we can get rid of `getDescriptionAllowingConversion` and return directly from here:

```suggestion
const getDescription = ( templateTitle: string ) =>
	sprintf(
		/* translators: %s is the template title */
		__(
			'Transform this template into multiple blocks so you can add, remove, reorder, and customize your %s template.',
			'woocommerce'
		),
		templateTitle
	);
```